kubeadm/v1.15.0/KubeCluster.ps1

```diff
@@ -0,0 +1,288 @@
+#
+# Copyright 2019 (c) Microsoft Corporation.
+# Licensed under the MIT license.
+#
+Param(
+    [parameter(Mandatory = $false,HelpMessage="Print the help")]
+    [switch] $help,
+    [parameter(Mandatory = $false,HelpMessage="Initialize Windows Control Plane node (unsupported)")]
+    [switch] $init,
+    [parameter(Mandatory = $false,HelpMessage="Install pre-requisites")]
+    [switch] $InstallPrerequisites,
+    [parameter(Mandatory = $false,HelpMessage="Join the windows node to the master")]
+    [switch] $join,
+    [parameter(Mandatory = $false,HelpMessage="Reset this windows node and cleanup everything")]
+    [switch] $reset,
+    [parameter(Mandatory = $false,HelpMessage="Path to input configuration json ")] 
+    $ConfigFile
+)
+
+function Usage()
+{
+    $bin = $PSCommandPath 
+    Get-Help $bin -Detailed
+
+    $usage = "
+    Usage: 
+		$bin [-help] [-init] [-join] [-reset]
+
+	Examples:
+        $bin -help                                                           print this help
+        $bin -InstallPrerequisites -ConfigFile kubecluster.json               Set up this Windows node to run containers
+        $bin -init -ConfigFile kubecluster.json                              joins the windows node to existing cluster control-plane (unsupported)
```

Comment:
Looks like a copy paste error, init should not work on Windows right?

Reply:
I said control-plane to say that this Windows node can't be a master and then added (unsupported). Kubeadm has `init` and also `join --experimental-control-plane` so I tried to cover both scenarios here. I can rephrase it to be more clear

Reply:
i think the `-init` flag can be indeed confusing, possibly omit completely for now.